src/lwc/geGiftEntryFormApp/geGiftEntryFormApp.js

```diff
@@ -51,16 +52,7 @@ const PAYMENT_AUTHORIZED_AT = DI_PAYMENT_AUTHORIZED_AT.fieldApiName;
 const DONATION_AMOUNT__C = DI_DONATION_AMOUNT_FIELD.fieldApiName;
 const DONATION_CAMPAIGN_NAME__C = DI_DONATION_CAMPAIGN_NAME_FIELD.fieldApiName;
 
-const PAYMENT_TRANSACTION_STATUS_ENUM = Object.freeze({
-    PENDING: 'PENDING',
-    AUTHORIZED: 'AUTHORIZED',
-    CANCELED: 'CANCELED',
-    CAPTURED: 'CAPTURED',
-    DECLINED: 'DECLINED',
-    NONRETRYABLEERROR: 'NONRETRYABLEERROR',
-    RETRYABLEERROR: 'RETRYABLEERROR',
-    REFUNDISSUED: 'REFUNDISSUED'
-});
+const PAYMENT_TRANSACTION_STATUS_ENUM = {};
```

Comment:
I'm not actually seeing an error in my console log, but shouldn't it be a TypeError to declare this as "const" and then attempt to assign it on line 80?